app/public/js/common/SCapiService.js

```diff
@@ -186,6 +186,86 @@ app.service('SCapiService', function($http, $window, $q, $log, $state, $statePar
     };
 
     /**
+     * Responsible to get track details
+     * @return {[object]} data response
+     */
+    this.getTrackDetails = function(songId) {
```

Comment:
No need to add new methods to call the SC api... you can just use the get method passing the `endpoint` and/or `params` that should work fine.

e.g
```
SCapiService.get('tracks/123')

SCapiService.get('tracks/123/comments', 'linked_partitioning=1&limit=30')
```